src/UseCases/GetPaymentID/GetPaymentIDUseCase.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\PaymentContext\UseCases\GetPaymentID;
+
+use WMDE\Fundraising\PaymentContext\Domain\Repositories\PaymentIDRepository;
+
+class GetPaymentIDUseCase {
+
+	private PaymentIDRepository $paymentIDRepository;
+
+	public function __construct( PaymentIDRepository $paymentIDRepository ) {
+		$this->paymentIDRepository = $paymentIDRepository;
+	}
+
+	public function getNewPaymentID(): int {
+		return $this->paymentIDRepository->getNewID();
+	}
+}
```

Comment:
You can drop this class and its test. Getting a payment id is an internal service of the payment domain, whereas "use cases" are the public API of a bounded context.